scalafmt-tests/src/test/resources/spaces/InterpolatedStringCurlyBracesAlways.stat

```diff
@@ -0,0 +1,21 @@
+spaces.inInterpolatedStringCurlyBraces = Always
+
+<<< Simple variable
```

Comment:
please create a single `.stat` file and set parameter in each specific test case. we stopped creating separate files for different combinations of parameters since there's now a way to adjust each case.